raster-foundry/raster-foundry
# Upgrade rollbar clients

## Overview

This PR upgrades all of our rollbar client to the MAX. The upgrades are EXTREME. Testing is gonna be INTENSE. Just kidding it's all pretty tame.

### Checklist


### Notes

Guess what doesn't have a changelog! If you guessed `ng-rollbar`, you win! However, their [initialization instructions](https://github.com/tandibar/ng-rollbar#initialize) still look like what we're doing, and the changelog from rollbar.js looked safe (there was a breaking change in October 2017 but I don't think it's something that will hurt us), so I think we're fine. It's tough to know if we're fine, but I _think_ we're fine.

## Testing Instructions

 * `docker-compose build batch`
 * `yarn install`
 * scala code should still compile
 * you should be able to throw an error in the `app-tasks` code and see that error, instead of an error that we're trying to do something wrong with rollbar
 * Check out the second to last commit
 * Bring up your servers and frontend
 * Watch your network traffic
 * List project exports
 * Check and see that you get a console error about a bad die roll
 * Look and see that you successfully posted to rollbar in the network tab

Closes azavea/raster-foundry-platform#419
